src/stateDirectives.js

```diff
@@ -244,8 +244,15 @@ function $StateRefActiveDirective($state, $stateParams, $interpolate) {
             state: state,
             params: newParams
           });
-          update();
```

Comment:
There, fresh air. 

We're looking at using either a single or a triple exclamation mark here to reach the desired effect - with a non-negating cast to boolean we're effectively setting lazy to true if the state was defined on initial load of the application, which is the reverse of what we want to do. 

So I'd say it's a matter of preference/taste now really; Single or Triple? I'm inclined to say triple for readability. 

poke @nateabele